function-images/tests/chained-function-serving/go.mod

```diff
@@ -0,0 +1,11 @@
+module tests/chained-functions-serving
```

Comment:
add this folder to .github/dependabot.yaml so that we can track its dependencies automatically